lib/turn.rb

```diff
@@ -0,0 +1,33 @@
+class Turn
+
+  attr_reader :action
+
+  def initialize
+    print "which row?\n > "
```

Comment:
puts "which row?"
print " >"

Is probably what I would have done. This is more efficient.